packages/data-models/constants.ts

```diff
@@ -48,6 +50,14 @@ const APP_LANGUAGES = {
   default: "gb_en",
 };
 
+/** Specifies the skins available to the current app build.
+ * "default": the skin used by default for a given build
+ * "available": an array of additional skins that are available to the current build */
+const APP_SKINS: { default: IAppSkin; available: IAppSkin[] } = {
+  default: { name: "default" },
+  available: [] as IAppSkin[],
```

Comment:
Thanks for adding the comment. Minor nit, as you have defined the `available` type above I expect should be fine to just leave `available: []` and ts should infer correctly